awpy/parser/parse_demo.go

```diff
@@ -568,7 +568,7 @@ func convertRoundMVPReason(r events.RoundMVPReason) string {
 	case events.MVPReasonBombPlanted:
 		return "MVPReasonBombPlanted"
 	default:
-		return unknown
+		return "Unknown"
```

Comment:
I think that should stay as `unkown` which is defined here: https://github.com/pnxenopoulos/awpy/blob/main/awpy/parser/parse_demo.go#L19

As one of the linters was complaining about it.